packages/web-components/src/components/link-list/link-list-item.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Link list item.
+ *
+ * @element dds-link-list-item
+ */
+@customElement(`${ddsPrefix}-link-list-item`)
+class DDSLinkListItem extends LitElement {
+  // eslint-disable-next-line class-methods-use-this
+  protected render() {
+    return html`
+      <slot @slotchange=${this.handleSlotChange} />
```

Comment:
```suggestion
      <slot @slotchange="${this._handleSlotChange}" />
```

And could you implement `._handleSlotchange()` private method to change the `style-scheme` attribute? It should replace `if (!this.querySelector('*')?.hasAttribute('style-scheme')) { this.querySelector('*')?.setAttribute('style-scheme', 'link-list'); }`.